gormigrate.go

```diff
@@ -164,6 +172,18 @@ func (g *Gormigrate) checkDuplicatedID() error {
 	return nil
 }
 
+func (g *Gormigrate) checkIDExist(migrationID string) error {
+	if migrationID == "" {
```

Comment:
Hmm... maybe we should remove this if and just return the error if `migrationID` is `""`?

I don't think `rollbackTo` should rollback all migration in any circumstance.

Reply:
I removed the check to allow empty migration ID. This also meant changes to `Migrate` so it doesn't migrate to an empty string, but to the last migration in the list, which I think is same semantics as before.